src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -59,13 +65,14 @@ iterated on the design.
 | 2. Create initial draft of specification     | [Complete][spec]             |
 | 3. Gather feedback and iterate on design     | Complete                     |
 | 4. Origin trial                              | Complete                     |
-| 5. **Launch**                                | Chrome 80, and beyond        |
+| 5. **Launch**                                | **Android: Chrome 80<br/>Edge for Windows: Chrome 85** |
```

Comment:
@petele & @jpmedley, just checking: Edge wants to be known as "Edge for Windows: Chrome 85"?

Reply:
To me it seems we should just refer to it as Edge 85. The fact that it's using Chromium underneath the hood is an implementation detail. I don't recall us using this phrasing for any of the other Chromium-based browsers. E.g. we don't say "Samsung Internet: Chrome 84"

Reply:
I wasn't thinking clearly. I had my week off on my mind.